pkg/scheduler/framework/plugins/estimator/estimator.go

```diff
@@ -0,0 +1,78 @@
+/*
+Copyright 2021 The Clusternet Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package estimator
+
+import (
+	"context"
+	"fmt"
+
+	appsapi "github.com/clusternet/clusternet/pkg/apis/apps/v1alpha1"
+	clsapi "github.com/clusternet/clusternet/pkg/apis/clusters/v1beta1"
+	scheduler "github.com/clusternet/clusternet/pkg/apis/scheduler"
+	framework "github.com/clusternet/clusternet/pkg/scheduler/framework/interfaces"
+	"github.com/clusternet/clusternet/pkg/scheduler/framework/plugins/names"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+// Estimator is a plugin than checks if a subscripthon need resources
+type Estimator struct {
```

Comment:
Here should be added to `pkg/scheduler/framework/plugins/defaultassigner/dynamic.go.